docs/pages/kubernetes-access/helm/guides/digitalocean.mdx

```diff
@@ -0,0 +1,190 @@
+---
+title: Get started with Teleport on DigitalOcean Kubernetes
+description: How to get started with Teleport on DigitalOcean Kubernetes
+---
+
+This guide will show you how to get started with Teleport on DigitalOcean Kubernetes. 
+
+## Prerequisites
+- DigitalOcean account.
+- Your workstation configured with [kubectl](https://kubernetes.io/docs/tasks/tools/), [Helm](https://helm.sh/docs/intro/install/), [doctl](https://docs.digitalocean.com/reference/doctl/how-to/install/), and the Teleport [tsh](https://goteleport.com/docs/installation/) client.
+
+## Step 1/4. Create a DigitalOcean Kubernetes cluster
+Create a new [DigitalOcean Kubernetes Cluster](https://cloud.digitalocean.com/kubernetes/clusters/)
+<Figure align="left" bordered caption="Create DigitalOcean Kubernetes cluster">
+  ![Create DigitalOcean Kubernetes cluster](../../../../img/helm/digitalocean/create-k8s.png)
+</Figure>
+
+<br />
+While the Kubernetes cluster is being provisioned, follow the "Getting Started" guide as shown below:
+<Figure align="left" bordered caption="Set up DigitalOcean Kubernetes client">
+  ![Set up DigitalOcean Kubernetes client](../../../../img/helm/digitalocean/setup-k8s.png)
+</Figure>
+
+
+## Step 2/4. Install Teleport 
+
+Add the Teleport repository to Helm.
+
+```code
+$ helm repo add teleport https://charts.releases.teleport.dev
+```
+
+Install Teleport in your Kubernetes cluster using the `teleport-cluster` Helm chart.
+```code
+$ CLUSTERNAME=tele.teleporters.dev # replace with your preferred domain name
+$ EMAIL_ADDR=dodemo@goteleport.com # replace with your email
+$ helm install teleport-cluster teleport/teleport-cluster --create-namespace --namespace=teleport-cluster \
+  --set clusterName=$CLUSTERNAME \
+  --set acme=true \
+  --set acmeEmail=$EMAIL_ADDR
+NAME: teleport-cluster
+LAST DEPLOYED: Tue Oct 26 17:01:21 2021
+NAMESPACE: teleport-cluster
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+```
+
+### Update DNS for `clusterName`
+First, get the external IP (from the `EXTERNAL-IP` field) for the Kubernetes cluster. 
+```code
+
+$ kubectl --namespace=teleport-cluster get services
+NAME               TYPE           CLUSTER-IP      EXTERNAL-IP       PORT(S)          AGE
+# teleport-cluster   LoadBalancer   10.245.163.12   192.168.200.200   443:31959/TCP,3023:30525/TCP,3026:30079/TCP,3024:32437/TCP   19m
+
+```
+Once you get the value for external IP (it may take a few minutes for this field to be populated), update your DNS record such that the clusterName's A record points to this IP address. For example `192.168.200.200` is the external IP in the above case.
```

Comment:
```suggestion
Once you get the value for the external IP (it may take a few minutes for this field to be populated), update your DNS record such that the clusterName's A record points to this IP address. For example `192.168.200.200` is the external IP in the above case.
```